include/ui/form.php

```diff
@@ -198,6 +198,12 @@ function bd_get_post_types() {
 	return array_merge( $builtin_types, $custom_types );
 }
 
+function bd_get_taxonomies(){
```

Comment:
If the function is going to only call `get_taxonomies` then directly use it. Don't create a new function for it.